
uint8 FIELD
#uint8 FIELD_PERSONAL = 1
#uint8 FIELD_DETECT = 2
#uint8 FIELD_WARN = 3
#uint8 FIELD_BRAKER = 4
#uint8 FIELD_POWER = 5
#uint8 FIELD_DEFAULT = 0